layouts/shortcodes/apm-ssi-uninstall-linux.md

```diff
@@ -0,0 +1,5 @@
+## Uninstall Single Step APM Instrumentation
```

Comment:
```suggestion
### Uninstall Single Step APM Instrumentation
```
To make sure this section doesn't get lost, I think we should make it a heading under Uninstall the Agent. This will help connect the sections more. Alternatively, we could make this an alert box instead of a section like this:

`<div class="alert alert-info">Text containing an <a href="https://google.com">HTML link.</a></div>`